turbo-repo/apps/app/src/lang/en.json

```
@@ -2633,6 +2633,142 @@
+      "viewer_prev": "Previous",
+      "viewer_next": "Next",
+      "viewer_close": "Close viewer",
+      "viewer_click_rename": "Click to rename",
+      "viewer_link_copied": "Link copied to clipboard",
+      "viewer_loading_doc": "Loading document...",
+      "viewer_load_error": "Unable to load document",
+      "viewer_change_category": "Change document type",
+      "category_change_loading": "Updating document type...",
+      "category_change_success": "Document type updated",
+      "category_change_error": "Failed to update document type",
+      "status_pending": "Pending",
```

Biome (2.4.15)

[error] The key status_pending was already declared.

(lint/suspicious/noDuplicateObjectKeys)